reports/performance.qmd

```diff
@@ -29,6 +29,13 @@ params:
 
 {{< include _outliers.qmd >}}
 
-{{< include _shap.qmd >}}
+```{r, results='asis'}
+# Dynamically render shap code based on whether they are enabled for model run
+# https://www.harveyl888.com/post/2022-05-12-knit_child_quarto/
+if (shap_exists) {
+  shap_qmd <- knitr::knit_child("_shap.qmd", quiet = T)
+  cat(shap_qmd, sep = "\n")
+}
+```
```

Comment:
This is how I came up with the full exclusion of the `_shap.qmd`. I tried for a long time to make some sort of conditional and `{{< include >}}` work, but I don't think quarto is quite there yet. With this solution we are able to remove all of the conditional r chunks in `_shap.qmd` which will make further developing are shap content a bit easier.